src/coreclr/vm/stackwalk.h

```diff
@@ -46,6 +46,7 @@ class AppDomain;
 
 #if defined(FEATURE_EH_FUNCLETS)
 #define RECORD_RESUMABLE_FRAME_SP
+#define PROCESS_EXPLICIT_FRAME_BEFORE_MANAGED_FRAME
```

Comment:
Is it necessary to move it here? I don't see any usage of this define outside of the stackwalk.cpp.

Reply:
I originally moved it because I tried to optimize some field usage based on it but that didn't quite pan out... That said, all the other definitions affecting the stack walk were already in this code block so it made sense to me to consolidate it.